components/properties/src/maps.rs

```diff
@@ -360,18 +366,27 @@ make_map_property! {
     ///
     /// ```
     /// use icu::properties::{maps, GeneralCategory};
-    /// use icu_collections::codepointtrie::CodePointTrie;
     ///
     /// let data =
-    ///     maps::load_general_category(&icu_testdata::unstable())
+    ///     maps::load_general_category(&icu_testdata::unstable_no_fallback())
     ///         .expect("The data should be valid");
     /// let gc = data.as_borrowed();
     /// assert_eq!(gc.get('木'), GeneralCategory::OtherLetter);  // U+6728
     /// assert_eq!(gc.get('🎃'), GeneralCategory::OtherSymbol);  // U+1F383 JACK-O-LANTERN
     /// ```
-    ///
-    /// [`CodePointTrie`]: icu_collections::codepointtrie::CodePointTrie
     pub fn load_general_category();
+    const:
+    /// Return a [`CodePointMapDataBorrowed`] for the General_Category Unicode enumerated property. See [`GeneralCategory`].
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use icu::properties::{maps, GeneralCategory};
+    ///
+    /// assert_eq!(maps::GENERAL_CATEGORY.get('木'), GeneralCategory::OtherLetter);  // U+6728
+    /// assert_eq!(maps::GENERAL_CATEGORY.get('🎃'), GeneralCategory::OtherSymbol);  // U+1F383 JACK-O-LANTERN
+    /// ```
+    pub const GENERAL_CATEGORY => SINGLETON_PROPS_GC_V1;
```

Comment:
Nit: I think I really would prefer these being functions instead of const variables, as in `maps::general_category()` instead of `maps::GENERAL_CATEGORY`